tests/Auth/AuthGuardTest.php

```diff
@@ -42,7 +42,6 @@ public function testBasicReturnsNullWhenAlreadyLoggedIn()
 
     /**
      * @expectedException \Symfony\Component\HttpKernel\Exception\UnauthorizedHttpException
```

Comment:
This backslash is also unnecessary, right?

Reply:
I find it not related to this PR ;)
If community will decide to drop it, let us drop it globally, not only in this one place.
Still, a lot of devs find it useful, benefits of `\` is especially visible with `\LogicException`, when you have also `LogicException` in namespace that is under tests and is already imported. Some ppl thinks that local `\Namespaced\LogicException extends \LogicException` would be used, but in fact `\LogicException` would be use

Reply:
Please don't remove it.

Yes, the test will still work correctly but IDEs usually treat references here like in code, which means they require proper qualification.

If you remove it, the IDE will signal they can't resolve (unless you use an explicit `use`-statement for that, etc.) which just annoys the group of developers enjoying their IDE while there's no visible benefit for any other group.

TL;DR: just migrate it to `setExpectedException` or leave it, thanks!

Reply:
not `setExpectedException` but `expectException`, and they don't want to migrate, see #22806 :(